docs/spec.md

```diff
@@ -5535,6 +5530,87 @@ Produces a `result` tuple from values `val`.
 // %result: ([1.0, 2.0], (3))
 ```
 
+### uniform_dequantize
+
+#### Semantics
+
+Performs element-wise conversion of quantized tensor `operand` to a
+floating-point tensor `result` according to the quantization parameters defined
+by the `operand` type.
+
+Formally, `result = dequantize(operand)`.
```

Comment:
Nit: the canonical way of expressing this in the document is "More formally, ...".

Reply:
done